jpsonic-main/src/main/java/com/tesshu/jpsonic/service/upnp/processor/UpnpDIDLFactory.java

```diff
@@ -183,6 +198,63 @@ private MimeType getMimeType(MediaFile song, Player player) {
         return mimeTypeString == null ? null : MimeType.valueOf(mimeTypeString);
     }
 
+    private StorageFolder createMusicFolder(int id, String name, ProcId procId, int childCount) {
```

Comment:
So far. . . There seems to be no problem with data flow. However, there may be a problem with image output. Confirmation required.

Reply:
ah. . This is my misunderstanding. When checking on the actual device, I made a mistake in setting the Base LAN URL. 🤫🤫🤫